litmus-portal/frontend/src/views/Observability/Overview/index.tsx

```diff
@@ -192,7 +192,13 @@ const Overview: React.FC = () => {
   let filteredDashboardListData;
   // Select the latest 3 dashboards
   if (monitoringDashboardCount > 0) {
-    filteredDashboardListData = dashboardListData?.ListDashboard.slice(-3);
+    filteredDashboardListData = dashboardListData?.ListDashboard.slice()
+      .sort(
```

Comment:
If there is some utility function in utils.ts for sorting based on time, that can be used too.